src/proxy/introspect.rs

```diff
@@ -0,0 +1,131 @@
+/*! <span class="stab portability" title="This is supported on crate feature `introspect` only"><code>introspect</code></span> Types returned by [`Proxy::introspect`].
+*/
+use super::{ProxyResult, Error, Proxy};
+use dbus_message_parser::{message::Message, value::Value};
+use serde_derive::Deserialize;
+use std::convert::TryInto;
+
+impl<'a> Proxy<'a> {
+    ///<span class="stab portability" title="This is supported on crate feature `introspect` only"><code>introspect</code></span>  Get a description of the object, including its interfaces (with signals and methods), objects below it in the object path tree, and its properties. See [D-Bus Specification](https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces-introspectable)
```

Comment:
Have to be this doc text a one liner? If not, split it to multiple lines, please.